.github/workflows/gardener-integration.yaml

```diff
@@ -0,0 +1,27 @@
+name: Gardener integration
+
+on:
+  push:
+    branches: [ "main" ]
+
+jobs:
+  gardener-integration-test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: save sa
+      shell: bash
+      run: 'echo $GARDENER_SA > /tmp/gardener-sa.yaml'
+      env:
+        GARDENER_SA: ${{ secrets.GARDENER_SA }}
+
+    - name: run test
+      run: make -C hack/ci gardener-integration-test
+      env:
+        IMG: europe-docker.pkg.dev/kyma-project/prod/serverless-operator:${{ github.sha }}
```

Comment:
Is it used because target `run-without-lm-on-cluster` has another target `deploy` which can override `IMG`:
```
.PHONY: deploy
deploy:
	@make -C ${PROJECT_ROOT} deploy \
  IMG=k3d-${REGISTRY_NAME}:${REGISTRY_PORT}/${OPERATOR_IMAGE_NAME}:${OPERATOR_IMAGE_TAG}
```

Reply:
we don't use the `deploy` target (look at the `deploy-cluster` one)